Utilities/Data.cpp

```diff
@@ -54,7 +54,7 @@ void FileManager::openFile(const std::string &fileName, const std::string &heade
       files[fileName] << header << "\n";
     }
   }
-  if (fileStates[fileName] == false) { // if file is closed ...
```

Comment:
I find the `(x == false)` syntax easier to read than (!x) because ! is so small and it is easier to read (!x) as (x) when x is a larger word such as here.

Reply:
Agreed.